proto/def/triggers.proto

```diff
@@ -39,6 +39,9 @@ message Broker {
 
   // broker name
   string name = 6;
+
+ // A comma separated list of host/port pairs to use for establishing the initial connection to the Kafka cluster.
+  string bootstrapServers = 7;
```

Comment:
More than a comma separated list, can you do directly a repeated string field here?

Reply:
On the data plane, Kafka configuration needs a comma-separated list so I thought it's more efficient to keep that as we need on the data plane avoiding allocating the string from the array for each consumer we instantiate.